<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qt_samp_01Class</class>
 <widget class="QMainWindow" name="qt_samp_01Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>基于QT的上位机开发示例（一）</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnConnect">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>连接</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDisconnect">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>80</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>断开连接</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>381</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>连接控制器</string>
    </property>
    <widget class="QLabel" name="Led_IsConnected">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>40</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Led1</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>381</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>使能伺服轴</string>
    </property>
    <widget class="QLabel" name="Led_isEnabled">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>40</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Led2</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnDisable">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>190</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>断开使能</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEnable">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>使能</string>
    </property>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>btnConnect</zorder>
   <zorder>btnDisconnect</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>btnDisable</zorder>
   <zorder>btnEnable</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qt_samp_01.qrc"/>
 </resources>
 <connections/>
</ui>
